backend/src/api/bitcoin/bitcoin-api.ts

```diff
@@ -41,7 +41,12 @@ class BitcoinApi implements AbstractBitcoinApi {
 
   $getBlockHeightTip(): Promise<number> {
     return this.bitcoindClient.getChainTips()
-      .then((result: IBitcoinApi.ChainTips[]) => result[0].height);
+      .then((result: IBitcoinApi.ChainTips[]) => {
+        const activeTip = result.find(tip => tip.status === 'active');
+        if (activeTip) {
+          return activeTip.height;
+        }
```

Comment:
Does `if (activeTip)` mean that it might be undefined? In that case we should return a default value.

Reply:
bitcoin core always has an active chaintip, right?
my IDE said it can be undefined because it doesn't know that

Reply:
i can add a non null assertion
like `return activeTip!.height;`

Reply:
Ok, I am just making sure to reduce any possibility of error because I am not sure if there will always be an active tip or not.

Reply:
src/rpc/blockchain.cpp:1783
`if (active_chain.Contains(block)) { status = "active" }` 

always 1 will be active since a block must be in a chain